model/types.go

```diff
@@ -16,12 +16,14 @@ limitations under the License.
 
 package model
 
+// Types - no idea what this is for - TODO(Dug)
```

Comment:
Should be able to drop this whole file I think, this is another relic from the prototype code.